internal/common/helpers.go

```diff
@@ -36,3 +38,31 @@ func IsStringInSortedSlice(slice []string, s string) bool {
 	}
 	return false
 }
+
+func FsSizeToUint64(size string) *uint64 {
+	// Read the number in the string
+	plain_number := regexp.MustCompile(`[[:digit:]]+`)
+	numbers_as_str := plain_number.FindAllString(size, 1)
+	number, err := strconv.ParseInt(numbers_as_str[0], 10, 64)
+	if err != nil {
+		return nil
+	}
+	return_size := uint64(number)
+
+	mega_byte := regexp.MustCompile(`[[:digit:]]+\s*(m|M)(b|B)?`)
```

Comment:
I suggest closely following the metric prefixes: https://en.wikipedia.org/wiki/Metric_prefix

Thus accepting only `M` as mega (`m` is milli), and `G` as giga (`g` doesn't exist).

As an addition, we can also accept binary prefixes: https://en.wikipedia.org/wiki/Binary_prefix

I would also vote against accepting `b`, so we don't confuse users with bits vs. bytes.

Reply:
This was in the request I got, but I might have incorrectly understand it.

Regardless, I agree that strictly following the prefixes would be the best solution, even though it might differ from some other tools (I don't exactly know what syntax kickstarts allow).

Reply:
I found something like this in the docs ( https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/installation_guide/sect-kickstart-syntax ):

> --size= - The size of the logical volume in MiB

I'm not a fan of this tbh.

btw e.g. `sfdisk` uses binary prefixes correctly (from `sfdisk(1)`): 

> The number may be followed by the multiplicative suffixes (KiB, MiB, GiB, TiB, PiB, EiB, ZiB and YiB), then it’s interpreted as size in bytes

Reply:
I think we can deviate from the request and do what we think makes more sense, so following the metric definitions sounds good to me.